src/main/java/com/redhat/cloud/custompolicies/app/rest/PolicyCrudService.java

```diff
@@ -434,11 +434,14 @@ public Response updatePolicy(@QueryParam ("dry") boolean dryRun, @PathParam("pol
 
           storedPolicy.populateFrom(policy);
           storedPolicy.customerid = user.getAccount();
+          storedPolicy.setMtimeToNow();
+          FullTrigger trigger = new FullTrigger(storedPolicy);
 
           existingTrigger.updateFromPolicy(storedPolicy);
```

Comment:
Is this not used anymore? What is it for? 
The `trigger` is not using the `updateFromPolicy`, so i assume we can remove that code?

Reply:
Yes, I think the `trigger` is rebase fallout. Will look at it later.